packages/client/screens/UserProfile/UserFollowersSearch/__tests__/UserFollowersSearch.test.tsx

```diff
@@ -0,0 +1,719 @@
+                                    invariant(
+                                        screen !== undefined,
+                                        'Screen must have been rendered before being in this state',
+                                    );
+
+                                    await waitFor(() => {
+                                        const followersFlatList =
+                                            screen.getByTestId(
+                                                `user-followers-search-flat-list`,
+                                            );
+                                        expect(followersFlatList).toBeTruthy();
+
+                                        const userCards =
+                                            within(
+                                                followersFlatList,
+                                            ).queryAllByTestId(/.*-user-card/);
+                                        expect(userCards.length).toBe(1);
+                                    });
+                                },
+                            },
+                        },
+
+                        'pressed my user card': {
+                            meta: {
+                                test: async ({ screen }: TestingContext) => {
+                                    invariant(
+                                        screen !== undefined,
+                                        'Screen must have been rendered before being in this state',
+                                    );
+
+                                    await waitFor(() => {
+                                        expect(
+                                            screen.getByTestId(
+                                                'default-my-profile-page-screen',
+                                            ),
+                                        ).toBeTruthy();
+                                    });
+                                },
+                            },
+                        },
+
+                        'pressed an other user card': {
+                            meta: {
+                                test: async ({ screen }: TestingContext) => {
+                                    invariant(
+                                        screen !== undefined,
+                                        'Screen must have been rendered before being in this state',
+                                    );
+
+                                    await waitFor(() => {
+                                        expect(
+                                            screen.getByTestId(
+                                                'default-profile-page-screen',
+                                            ),
+                                        ).toBeTruthy();
+                                    });
+                                },
+                            },
+                        },
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think this state should be out of `User relations are viewable` and `User found` states as when we transition to it, the `meta.test` of its parents will be called. It works because React Navigation keeps all the screens in memory, but this is an implementation detail.
If it works only because of this implementation, it means that the model is not totally accurate.
This is just a thought I wanted to share with you, but I require absolutely no change.

Reply:
Indeed :open_mouth: 
About to check what's possible

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: